utils/contentful-data-clearer/clear-db-script.sql

```diff
@@ -0,0 +1,33 @@
+DELETE FROM [Contentful].[Answers]
+DELETE FROM [Contentful].[Buttons]
+DELETE FROM [Contentful].[ButtonWithEntryReferences]
+DELETE FROM [Contentful].[ButtonWithLinks]
+DELETE FROM [Contentful].[Categories]
+DELETE FROM [Contentful].[Checkboxes]
+DELETE FROM [Contentful].[CheckboxOptions]
+DELETE FROM [Contentful].[ComponentDropDowns]
+DELETE FROM [Contentful].[ContentComponents]
+DELETE FROM [Contentful].[Headers]
+DELETE FROM [Contentful].[InsetTexts]
+DELETE FROM [Contentful].[NavigationLink]
+DELETE FROM [Contentful].[Pages]
```

Comment:
Another option that avoids the relationship problem is to turn off foreign key checks, get all tables from information schema with contentful schema, truncate them and turn key checks back on, future proofing

Reply:
A good shout mate.

I'll stick a "ticket" about it on the dev board we made yesterday, so it can be added to the backlog at some point.

Reply:
cheers, ill give this the go then